docs/README.md

```diff
@@ -598,7 +600,7 @@ This completes the most recently started migration.
 
 Running `pgroll complete` when there is no migration in progress is a no-op.
 
-Completing a `pgroll` migration removes the previous schema version from the database, leaving only the latest version of the schema.
+Completing a `pgroll` migration removes the previous schema version from the database (e.g. `public_02_create_table`), leaving only the latest version of the schema (e.g. `public_03_add_column`), available under both specific and main schema name (i.e. both `public_03_add_column` and `public`).
```

Comment:
Another attempt to make it clear that the version schema contain views and the base schema (default `public`) contains the tables.

```suggestion
Completing a `pgroll` migration removes the previous schema version from the database (e.g. `public_02_create_table`), leaving only the latest version of the schema (e.g. `public_03_add_column`). At this point, any temporary columns and triggers created on the affected tables in the `public` schema will also be cleaned up, leaving the table schema in its final state.
```